org.eclipse.jdt.ls.core/src/org/eclipse/jdt/ls/core/internal/managers/ProjectsManager.java

```diff
@@ -251,6 +251,9 @@ public void fileChanged(String uriString, CHANGE_TYPE changeType) {
 			return;
 		}
 		IResource resource = JDTUtils.isFolder(uriString) ? JDTUtils.findFolder(uriString) : JDTUtils.findFile(uriString);
+		if (resource == null && changeType == CHANGE_TYPE.CREATED) {
```

Comment:
I believe it should be moved to the `if (isBuildFile(..)` block, L283

Reply:
so this doesn't work if you want to create a new module within an existing maven structure

Reply:
> I believe it should be moved to the if (isBuildFile(..) block, L283

We will not come to L283 if a resource is null